src/components/base/outline-accordion/outline-accordion.ts

```diff
@@ -0,0 +1,116 @@
+import { html, TemplateResult, CSSResultGroup } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { OutlineElement } from '../outline-element/outline-element';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import componentStyles from './outline-accordion.css.lit';
+
+export type AccordionPanel = {
+  heading: string;
+  // TODO: Discuss removing "@typescript-eslint/no-explicit-any" rule. Potentially change to content: any.
+  content: unknown;
+};
+
+/**
+ * Accordion Component
+ * @element outline-accordion
+ */
+@customElement('outline-accordion')
+export class OutlineAccordion extends OutlineElement {
+  static styles: CSSResultGroup = [componentStyles];
+
+  /**
+   * The data aray of <AccordionPanel> objects.
+   * AccordionPanel.content is of indeterminate type, and is passed into the panel <slot>.
+   */
+  @property({ type: Array }) data!: AccordionPanel[];
+  /**
+   * Set to 'true' to create an accordion that can only have one panel open ata a time.
+   */
```

Comment:
I LOVE all of the code comments in here!

Styling nitpick: I assume that the coding standards for Outline require a blank line above multiline comments like this? Did this pass linting? Also, `ata` should be `at a`.

Reply:
Yeah it passed linting. @mabry1985 gets a lot of the credit as I'm straight mimicking his comment style because it does look good.